src/actions/actions.ts

```diff
@@ -368,6 +368,26 @@ export class ActionPayloadTypes {
         cursor: string
     } = z
 
+    /**
+     * Fetch number of table records
+     * 
+     * @param bcName Business component that initiated data fetch
+     */
+    bcFetchRecordCounter: {
+        bcName: string,
```

Comment:
We should prefer `widgetName` instead of `bcName` when possible.  
For example, here we have an action that requires records counting - we request it from some widget, and there are  cases when it's important to know which specific widget was a source of this request, because if there are several widgets sharing the same business component there is really no way to tell them apart. So it's better to pass the widget name in actions payloads and then derive bcName from this widget.

Use bcName only in situations when there is no initiating widget